HIP/hip-904-frictionless-airdrop.md

```diff
@@ -464,34 +770,91 @@ Scenario 2: If Dave submits a TokenClaim for token T5 and a Token claim of seria
 
 `automatic_token_associations` containing
 
-- Daves automatic association with token T5
-- Dave automatic association with token T6
+- Dave's automatic association with token T5
+- Dave's automatic association with token T6
 
 `pending_airdrops` will be empty
 
-Note: NFT serial 6 of T6 is left in pending airdrop, but all of the pending fungible amount of T5 was transferred 
+Note: NFT serial 6 of T6 is left in pending airdrop, but all of the pending fungible amount of T5
+was transferred.
 
-Scenario 3: Eric submits a TokenCancelAirdrop for token T7 and a TokenCancelAirdrop of serial 7 of T8
+Scenario 3: Alice submits a TokenCancelAirdrop for token T7 and a TokenCancelAirdrop of serial 7 of T8
 
 `tokenTransferLists` will be empty
 
 `automatic_token_associations` will be empty
 
 `pending_airdrops` will be empty, details are in transaction body
 
-Note: NFT serial 6 of T6 is left in pending airdrop, but all of the pending fungible amount of T5 was transferred 
+Note: NFT serial 8 of T8 is left in pending airdrop, but all of the pending fungible amount of T7
+was transferred.
 
 #### Reject
 
+> REVIEW NOTE
+>> Do we actually need the `nft_ids` field?  Is reject a rejection of the token type, or do we
+>> truly intend to allow rejecting a single NFT?
```

Comment:
I believe it would be good to have a fine grain control over which exact NFT you want to reject, since there might be a mistake in a specific NFT, not the whole unique token collection.

Reply:
It's a recurring confusion, but `tokenReject` has very little connection with airdrops.

Reject is not a function of a sender, and is not a remedy for a mistaken airdrop.

Reject is a mechanism for an account holder to completely dissociate from a token type without paying, possibly unreasonable, custom fees.  A single NFT, on the other hand, can be transferred via normal crypto transfer transactions if an account holder merely wishes to dispose of it.

Reply:
Agree on that. The only thing that I want to add is, large custom fees could be also applied on a simple transfer of a NFT for a single serial number itself.